crates/turborepo-lib/src/engine/builder.rs

```diff
@@ -389,6 +393,21 @@ impl Error {
     }
 }
 
+const INVALID_TOKENS: &[&str] = &["$colon$", "/", "\\"];
```

Comment:
should we just ban dollar signs in general or require that it match `a-zA-Z:-_` or something? And if we're going down that route, what about other languages/unicode/etc?

Reply:
Per comment from @chris-olszewski, I'm going to try to minimize what we ban, and just go for the one escape sequence we use.